test/components/componentTests.ts

```diff
@@ -1,115 +1,187 @@
 ///<reference path="../testReference.ts" />
 
-function assertComponentXY(component: Plottable.Component, x: number, y: number, message: string) {
-  // use <any> to examine the private variables
-  let translate = d3.transform((<any> component)._element.attr("transform")).translate;
-  let xActual = translate[0];
-  let yActual = translate[1];
-  assert.strictEqual(xActual, x, "X: " + message);
-  assert.strictEqual(yActual, y, "Y: " + message);
-}
-
-describe("Component behavior", () => {
-  let svg: d3.Selection<void>;
-  let c: Plottable.Component;
+describe("Component", () => {
   let SVG_WIDTH = 400;
   let SVG_HEIGHT = 300;
-  beforeEach(() => {
-    svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-    c = new Plottable.Component();
-  });
 
-  describe("anchor()", () => {
-    it("anchor()-ing works as expected", () => {
+  describe("anchoring", () => {
+
```

Comment:
really minor, I don't like the blank lines after each describe, and as Andrei pointed out, describe title are usually capitalized  
not a blocker tho
+1 as is